Mathlib/Algebra/Lie/TraceForm.lean

```diff
@@ -338,21 +338,17 @@ lemma killingForm_eq_zero_of_mem_zeroRoot_mem_posFitting
   LieModule.eq_zero_of_mem_weightSpace_mem_posFitting R H L
     (fun x y z ↦ LieModule.traceForm_apply_lie_apply' R L L x y z) hx₀ hx₁
 
+lemma killingForm_lieInvariant : (killingForm R L).lieInvariant L := by
+  intro x y z
+  rw [← lie_skew, map_neg, LinearMap.neg_apply, LieModule.traceForm_apply_lie_apply R L L]
```

Comment:
This is just:
```suggestion
lemma killingForm_lieInvariant : (killingForm R L).lieInvariant L :=
  LieModule.traceForm_apply_lie_apply' R L L
```
Probably we should just have `tracefForm_lieInvariant` next to `LieModule.traceForm_apply_lie_apply'` instead.